packages/coinstac-ui/app/render/components/maps/maps-edit-form.jsx

```diff
@@ -30,15 +49,21 @@ const styles = theme => ({
 });
 
 function MapsEditForm({
-  consortiumId, pipeline, dataMap, onSubmit, onChange, saved, classes,
+  consortiumId, pipeline, dataMap, onSubmit, onChange, saved, classes, error,
 }, { router }) {
   const handleGoBackToConsortium = () => {
     localStorage.setItem('HIGHLIGHT_CONSORTIUM', consortiumId);
     router.push('/dashboard/consortia');
   };
 
+  const pickerviews = [];
+  const fieldviews = [];
+
+  const dataCount = Object.entries(dataMap).length;
+
   return (
     <form onSubmit={onSubmit}>
+      {dataCount > 0 && !error && (
```

Comment:
how does all this work with comps that dont need mapping?

Reply:
you wouldn't get to this stage. In the maps list page you'd see that the comp is already mapped, like so:
![Screenshot from 2021-12-17 15-48-46](https://user-images.githubusercontent.com/37052303/146593532-a0c7e70a-4724-411c-bed7-1be8e32108d5.png)

the details page would go like this
![Screenshot from 2021-12-17 15-48-50](https://user-images.githubusercontent.com/37052303/146593567-fbc2ba2a-a8cb-43a5-9134-55217711ceab.png)